### PR TITLE
Update the sales website (#3814)

### LINKED ISSUE #3814: Update the Sales website
Update the Sales website

### PR DESCRIPTION
**Pull request by msynk**
this closes #3814

### CODE REVIEW

File: src/Websites/Sales/Bit.Websites.Sales/Web/compilerconfig.json.defaults [hunk 1]
```diff
@@ -6,7 +6,8 @@
             "style": "expanded",
             "sourceMapUrls": "relative",
             "relativeUrls": true,
-            "sourceMap": false
+            "sourceMap": false,
+            "useNodeSass": false
```
Comment (ysmoradi) [inline src/Websites/Sales/Bit.Websites.Sales/Web/compilerconfig.json.defaults:10]: What's that `useNodeSass`?

File: src/Websites/Sales/Bit.Websites.Sales/Api/appsettings.json [hunk 2]
```diff
@@ -36,11 +36,15 @@
             "Password": null
         },
         "HealthCheckSettings": {
-            "EnableHealthChecks": true
+            "EnableHealthChecks": false
```
Comment (ysmoradi) [inline src/Websites/Sales/Bit.Websites.Sales/Api/appsettings.json:39]: We can keep this true

File: src/Websites/Sales/Directory.Build.props [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <PropertyGroup>
         <BlazorMode>BlazorServer</BlazorMode>
         <!-- You can use either BlazorServer or BlazorWebAssembly or BlazorHybrid -->
-        <WebAppDeploymentType>DefaultDeploymentType</WebAppDeploymentType>
+        <WebAppDeploymentType>SSR</WebAppDeploymentType>
```
Comment (ysmoradi) [inline src/Websites/Sales/Directory.Build.props:6]: Revert this

**Review by ysmoradi — CHANGES_REQUESTED** [review]

File: src/Websites/Sales/Bit.Websites.Sales/Web/compilerconfig.json.defaults (hunk 1 above)
Comment (msynk) [reply src/Websites/Sales/Bit.Websites.Sales/Web/compilerconfig.json.defaults:10]: it's probably from the webcompiler extension.

File: src/Websites/Sales/Directory.Build.props (hunk 3 above)
Comment (msynk) [reply src/Websites/Sales/Directory.Build.props:6]: we talked about it and decided to go with SSR from now on.

**Review by ysmoradi — APPROVED** [review]

**Pull request merged**